reverse_engineering/helpers/getSchemaSequences.js

```diff
@@ -145,24 +160,27 @@ const getSchemaSequenceDtos = async ({ schema, execute, logger }) => {
 };
 
 /**
- * @param {{ schema: string, execute: Execute, logger: object }}
+ * @param {{ schema: string, namesOfSequences: string[], execute: Execute, logger: object }}
  * @returns {Promise<{ [sequenceName: string]: string }>}
  */
-const getSchemaSequenceDdl = async ({ schema, execute, logger }) => {
+const getSchemaSequenceDdl = async ({ schema, namesOfSequences, execute, logger }) => {
 	try {
 		logger.log('info', { message: 'Start getting sequences DDL' }, 'Getting sequences');
 
-		const sequenceDdlScriptsPromise = execute(
-			`
-      SELECT JSON_OBJECT(
-          'sequenceName' VALUE OBJECT_NAME,
-          'ddlScript'    VALUE DBMS_METADATA.GET_DDL(OBJECT_TYPE, OBJECT_NAME, OWNER)
-      )
-      FROM ALL_OBJECTS
-      WHERE OBJECT_TYPE = 'SEQUENCE'
-      AND OWNER = '${schema}'
-      `,
-		);
+		const sequenceDdlScriptsPromise = mapLimit(namesOfSequences, 50, async sequenceName => {
+			try {
+				const sequenceDdlScript = await execute(
+					`SELECT JSON_OBJECT('ddl' VALUE DBMS_METADATA.GET_DDL('SEQUENCE', '${sequenceName}', '${schema}')) FROM DUAL`,
```

Comment:
@Vitalii4as FROM DUAL is new (it looks like), was it missing previously or you make sure that Duality views are properly handled explicitly?